repository/t.json

```diff
@@ -2375,19 +2375,23 @@
 			]
 		},
 		{
-			"name": "Tint",
-			"details": "https://github.com/yyyc514/Tint",
-			"author": "jgoebel",
+			"name": "Tint Terminal",
+			"previous_names": ["Tint"],
+			"details": "https://github.com/joshgoebel/sublime-tint-terminal",
+			"author": "joshgoebel",
```

Comment:
Since the author here is just your GitHub username, you can leave it out.

Reply:
I see a lot of people using their liternal name perhaps I should change it to "Josh Goebel" then?

Reply:
Yes, you can also do that 👍🏻

Reply:
Better?

Reply:
👍🏻